tests/functional/lti_certification/v13/core/test_bad_payloads.py

```
@@ -0,0 +1,134 @@
+import pytest
+
+
+@pytest.mark.xfail(cause="Work in progress")
+class TestBadPayloads:
+    """
+    The first few tests will be those that are in one or another way known to be invalid for 1.3 Core Launches.
+
+    http://www.imsproject.org/spec/lti/v1p3/cert/#known-bad-payloads
+    """
+
+    def test_no_kid_sent_in_jwt_header(
+        self, jwt_headers, make_jwt, test_payload, do_lti_launch
+    ):
+        """
+        The KID is missing from the header of the JWT (preventing the verification of the signing of the JWT)
+        """
+        del jwt_headers["kid"]
+
+        response = do_lti_launch(
+            {"id_token": make_jwt(test_payload, jwt_headers)}, status=403
+        )
+
+        assert response.html
+
+    @pytest.mark.xfail(reason="JWT verification disabled")
```

Double fail. Fail harder!